tests/system/migrations/test_0032_fix_motion_origin_relations.py

```diff
@@ -129,3 +129,105 @@ def test_all_origin_ids(write, finalize, assert_model):
         },
         position=3,
     )
+
+
+def test_deleted_created_motions(read_model, write, finalize, assert_model):
+    setup_data(write)
+    write(
+        {
+            "type": "delete",
+            "fqid": "motion/2",
+        },
+        {
+            "type": "update",
+            "fqid": "motion/1",
+            "fields": {"derived_motion_ids": [3]},
```

Comment:
This is wrong, `motion/3` is not created yet

Reply:
changed